Tools/autotest/sim_vehicle.py

```diff
@@ -348,6 +349,12 @@ def wait_unlimited():
         "waf_target": "bin/ardurover",
         "default_params_filename": "default_params/rover-skid.parm",
     },
+    # SUB
+    "vectored": {
```

Comment:
better to have sub or sub-vectored to better identification and allow other sub type !